vendor/js/jquery-form/jquery.form.js

```diff
@@ -53,7 +53,7 @@ $.fn.ajaxSubmit = function(options) {
 	if (typeof options == 'function')
 		options = { success: options };
 
-	var url = $.trim(this.attr('action'));
+	var url = this.attr('action').trim();
```

Comment:
I was wondering why don't we check js files in this folder while running npm run lint:js